sw/host/opentitanlib/src/transport/cw310/usb.rs

```diff
@@ -334,7 +347,7 @@ impl Backend {
         bitstream: &[u8],
         progress: Option<&dyn Fn(u32, u32)>,
     ) -> Result<()> {
-        self.fpga_erase()?;
+        self.fpga_prepare(20_000_000)?;
```

Comment:
Nit. Should the speed be a constant?

Reply:
I agree, it would be nice for readability:
```suggestion
        const PROG_SPEED : u32 = 20_000_000;
        self.fpga_prepare(PROG_SPEED)?;
```

Reply:
While we're bikeshedding, I would prefer `PROG_SPEED_HZ` :)

Reply:
OK, I won't suggest defining a [newtype](https://rust-unofficial.github.io/patterns/patterns/behavioural/newtype.html#motivation) for Hz :) 

_P.S. Seriously, a `const` is good enough. I just wanted to mention this pattern since we are talking about units._

Reply:
Lol, yes, we must have an `impl From<Hz> for KHz` and `impl From<KHz> for MHz`, so then we can say `MHz{20}.into().into()`.

`</joke>`